### ISSUE TITLE
xtriggers

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#131]
The new UI should display xtriggers at least in the graph view.

We have the potential to go further with xtrigger support, see also

* https://github.com/cylc/cylc-flow/issues/3221

**Comment by hjoliver** [comment]
(The old GUI does show xtriggers in the Graph View).

**Comment by hjoliver** [comment]
> Should queues use option (1) or (2) or something else?

Shouldn't be too messy if queue "triggers" are only shown when the task is actively held back - i.e. ready to submit as soon as released by the queue.

**Comment by oliver-sanders** [comment]
True, perhaps we should add logic to Cylc Flow for this special case.

However it is common to see suites (e.g. rose stem) where almost all tasks would be held back by a queue.

**Comment by oliver-sanders** [comment]
@hjoliver one for the workshop.

**Comment by oliver-sanders** [comment]
[cylc-con-2020]

* We are happy with this
* However, queues are special, they should appear next to the task icon in the space where the held modifier is located.
* Xtriggers should remain after they succeed but their icon should be replaced with a tick or succeeded icon.
